### ISSUE TITLE
Project logo, public foreground and background

### ISSUE THREAD

**Issue by scott-lucius-space** [tractr/directus-sync#8]
Any suggestions on an easy way to null out specific fields via configuration?

Specifically looking at:

```
    "project_logo": null,
    "public_foreground": null,
    "public_background": null,
```

in the `settings.json` collection file which were in my case set to UUIDs from the media library. Or are you considering situations like this with the custom data backup you mentioned in Discord? Thanks for such a great tool!

**Comment by EdouardDem** [comment]
@scott-lucius-space Thank you.
That's a good point. Ultimately I would like to also synchronise the assets referenced by the settngs.
Meanwhile, we could just ignore these fields. There is no point to syncrhonise the values of the fields without the assets.
I will push a quickfix to discard these 3 fields.

**Comment by EdouardDem** [comment]
> Or are you considering situations like this with the custom data backup you mentioned in Discord?

Yes I'm working on an advanced configuration that will allow this.

**Comment by EdouardDem** [comment]
I've done a PR #11 . I will test this tomorrow morning and merge it.

**Comment by EdouardDem** [comment]
The PR has been merged
